/**:
  ros__parameters:
    use_operation_mode_availability: true
    rate: 10.0
    input_qos_depth: 1000
    graph_qos_depth: 1
